commands/Avatar.js

```diff
@@ -0,0 +1,43 @@
+import Command from "./base/Command.js";
+import { findMembers, findUsers } from "../utils/finderUtil.js";
+
+export default class Avatar extends Command {
+  constructor() {
+    super({
+      name: "avatar",
+      category: "tool",
+      description: "Returns aforementioned avatar"
+    });
+  }
+
+  run(client, message, args) {
+    // Joins the whole argument onto spaces
+    const messageArgs = args.join(" ");
+    let member;
+
+    if (args.length) {
+      // Uses the user query
+      [member] =
+        findMembers(messageArgs, message.guild) ||
+        findUsers(messageArgs, client);
```

Comment:
this won't work as intended, arrays are always true
![image](https://user-images.githubusercontent.com/8843386/78421136-b9386500-7655-11ea-9f2d-8e88423bad2a.png)
You can however just grab the first element and check on that, javascript won't throw 
![image](https://user-images.githubusercontent.com/8843386/78421159-d4a37000-7655-11ea-86e4-2849a3c193a7.png)

```suggestion
      member =
        findMembers(messageArgs, message.guild)[0] ||
        findUsers(messageArgs, client)[0];
```

Reply:
On my way to fix this~!